capstone/src/test/java/com/google/step/clients/RestaurantClientTest.java

```diff
@@ -33,7 +34,7 @@
 
 /** Test class for the Restaurant.java class */
 @RunWith(JUnit4.class)
-public final class RestaurantTest {
+public final class RestaurantClientTest {
```

Comment:
You don't seem to be using the Datastore helper class, which could be a good way of testing that the DAO behaves in an expected way. Perhaps you could add the simple consistency tests that Martin had in `LoginServlet.java`. And in terms of testing that things are working the way you expect, you could maybe add an entity to the Datastore and retrieve it by ID or something.